software/harbor.yml

```diff
@@ -0,0 +1,11 @@
+name: "Harbor"
+website_url: "https://goharbor.io/"
+source_code_url: "https://github.com/goharbor/harbor"
+description: "Cloud native image registry that stores, signs, and scans content."
+licenses:
+  - Apache-2.0
+platforms:
+  - Docker
+  - K8S
+tags:
+  - Software Development - Project Management
```

Comment:
I don't think this category is correct

Reply:
erm, what should be the proper one 🤔

Reply:
should it be File Transfer - Object Storage & File Servers, similar to ZOT oci registry?

Reply:
I adjusted it to ZOT's tag, as that should be OK, pending further thoughts from you nodiscc and other collaborators.